### ISSUE TITLE
Ctrl + Space doesn't open autocomplete dropdown in GraphQL

### ISSUE THREAD

**Issue by itsfabijano** [getinsomnia/insomnia#5692]
### Expected Behavior

Ctrl (^) + Space should open autocomplete dropdown.
![image](https://user-images.githubusercontent.com/16303843/212890516-3e77b218-1aac-49f9-b247-2ef89fe24b34.png)

### Actual Behavior

Ctrl (^) + Space has no effect at all.

### Reproduction Steps

1. Create a new GraphQL request
2. Enter a valid api url like https://api.spacex.land/graphql/
3. Start typing a query
```graphql
query q {
	launchesPast(limit: 10) {
		id

	}
}
```
4. Put the curser under the `id` attribute
5. Hit Control + Space 

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

2022.7.3

### What operating system are you using?

macOS

### Operating System Version

maxOS Monterey 12.6.1

### Installation method

download from https://insomnia.rest/

### Last Known Working Insomnia version

2022.6.0

**Comment by filfreire** [comment]
@itsfabijano I can also reproduce this in a way in [latest 2022.7.5](https://github.com/Kong/insomnia/releases/tag/core%402022.7.5).

I do get autocomplete suggestions while writing:
![image](https://user-images.githubusercontent.com/11976836/213692231-9c490fde-70cb-4db3-a20f-a11a6d592931.png)

But seems the auto-complete keyboard shortcut doesn't trigger these same suggestions.
Only seems to work on the query space:
![image](https://user-images.githubusercontent.com/11976836/213692511-9bc67c3a-54fc-40ce-b565-e820693ffac4.png)

**Comment by emil-nasso** [comment]
I have this, but it behaves in a weird way for me.

* I write a part of a query and press enter for a new line.
* The cursor gets indented.
* If i press ctrl + space, nothing happens.
* If I only press space, the cursor moves forward one space and I get the complete autocomplete.

A workaround is to add an extra space whenever you want to autocomplete and clean up the query when you are done.

If I write a letter, I also get autocomplete (for that letter, obviously) but if I press ESC (to hide the autocomplete) and then ctrl + space, the autocomplete doesn't reappear.

I have tried to reset the autocomplete keyboard shortcut but that didn't help.

TL;DR: Autocomplete works and is triggered by typing, including a space to get all autocomplete suggestions but the ctrl + space shortcut seems to be broken.

**Comment by EvilVir** [comment]
Same here :(

**Comment by itsfabijano** [comment]
@filfreire It's still happening in the latest stable version [latest 2022.7.5](https://github.com/Kong/insomnia/releases/tag/core%402022.7.5) and also in the newest beta [2023.1.0-beta.3](https://github.com/Kong/insomnia/releases/tag/core%402023.1.0-beta.3).

**Comment by RammsteinAM** [comment]
This is very frustrating.
![Sadge (1)](https://user-images.githubusercontent.com/481267/222659951-777a83fc-604a-4047-8ed9-1479fa7cc009.png)

Also when I write "query", open curly braces and press Enter, instead of a line break, Insomnia autofills the first query name. This is very inconvenient.

As a workaround I'm trying to use Space for the suggestions instead of Ctrl + Space, avoid pressing Enter, and use Ctrl + Shift + I to format the code several times during input. But it's not easy to get used to it and I don't even want to.

**Comment by itsfabijano** [comment]
For me the only real workaround is to use this version [2022.6.0](https://github.com/Kong/insomnia/releases/tag/core%402022.6.0). All newer versions are difficult to use with GraphQL.

**Comment by flan6** [comment]
> For me the only real workaround is to use this version [2022.6.0](https://github.com/Kong/insomnia/releases/tag/core%402022.6.0). All newer versions are difficult to use with GraphQL.

this version  2022.6.0 didnt work for me either. Same behavior, ctrl+space does nothing. even tried some older version 2022.5.1 and none worked as intended.


OS arch linux

**Comment by SamuelToh** [comment]
+1

This is frustrating, feels like a downgrade to me.

**Comment by oysandvik94** [comment]
Incredibly annoying, I have to run all my queries through a prettifier afterwards because I'm using space to trigger the autocomplete.

**Comment by itsfabijano** [comment]
Any news here? I would like to update...
